.pylintrc

```diff
@@ -40,6 +40,7 @@ disable=too-many-ancestors,
 	too-many-arguments,
 	too-many-locals,
 	too-many-instance-attributes,
+	too-many-statements,
```

Comment:
Can we remove `too-many-statements` from here before merging?

Reply:
Yes  I'll remove this later